python/src/aiconfig/editor/client/src/components/SettingsPropertyRenderer.tsx

```diff
@@ -367,8 +398,22 @@ export default function SettingsPropertyRenderer({
             <Stack>{subpropertyControls}</Stack>
           </>
         );
+      } else {
+        propertyControl = (
+          <div style={{width: "100%"}}>
+          <Stack>
+            <PropertyLabel
+             propertyName={propertyName}
+             propertyDescription={propertyDescription}
+           />
+            <JSONEditor
+              content={initialValue as JSONObject}
+              onChangeContent={setAndPropagateValue}
+            />
+          </Stack>
+          </div>
```

Comment:
```suggestion
          <Stack>
            <PropertyLabel
             propertyName={propertyName}
             propertyDescription={propertyDescription}
           />
           <div style={{width: "100%"}}>
              <JSONEditor
                content={initialValue as JSONObject}
                onChangeContent={setAndPropagateValue}
              />
            </div>
          </Stack>
```
Let's wrap the div around the JSONEditor only. If that still doesn't make it wider can you try {{minWidth: "350px}} or something like that? We have a min width of 400px for the settings container (https://github.com/lastmile-ai/aiconfig/blob/3b91a530a376faf13b2ae7ad3bf0ba1b20114548/python/src/aiconfig/editor/client/src/components/prompt/PromptActionBar.tsx#L48)